CKB-Explorer-PRDs/Glossary/Ongoing.md

```diff
@@ -0,0 +1,54 @@
+| 区块详情页 | Uncle Count | The number of [uncle blocks](https://docs.nervos.org/docs/basics/glossary/#uncle) in the current block. | checked(sven) | - |
+| 区块详情页 | Miner | A miner is a computer that provides computing power to validate transactions and create the blocks in the blockchain. |checked(sven) | https://docs-old.nervos.org/glossary/glossary-general#miner |
+| 区块详情页 | Miner Message | 矿工对当前区块的留言。 | checked(sven) | - |
+| 区块详情页 | Epoch | In CKB, an Epoch is a fixed period of time during which the difficulty level for Proof of Work (PoW) mining is held constant. The duration of an Epoch is set to 4 hours, and all the blocks in the same Epoch share the same difficulty target. The difficulty adjustment algorithm aims to stabilize the orphan block rate at 2.5% by adjusting the difficulty level for each Epoch based on the average block time of the previous Epoch. The Epoch is used to ensure that the network remains secure and that miners are incentivized to continue mining the network. The Epoch concept is used in many other blockchain networks as well, and is an important component of the consensus mechanism. | checked(sven) | [CKB doc](https://docs.nervos.org/docs/basics/glossary#epoch) |
+| 区块详情页 | Epoch Start Number | In CKB, Epoch Start Number refers to the block number at which a new Epoch begins. The Epoch Start Number is calculated based on the number of blocks that have been mined since the last Epoch started. Specifically, the Epoch Start Number is equal to the current block number minus the remainder of the current block number divided by the Epoch length. For example, if the current block number is 1000 and the Epoch length is 180 blocks, then the Epoch Start Number would be 1000 - (1000 % 180) = 940. The Epoch Start Number is used to determine when the difficulty level for Proof of Work (PoW) mining should be adjusted, and is an important component of the consensus mechanism in CKB. | checked(sven) | - |
+| 区块详情页 | Block Index | In CKB, Block Index refers to the numerical index assigned to a block in the blockchain. The Block Index is a unique identifier for each block in the chain, and is used to order the blocks in the blockchain. The first block in the chain, also known as the Genesis Block, has a Block Index of 0, and each subsequent block is assigned the next sequential index. The Block Index is an important component of the blockchain, as it is used to maintain the integrity and security of the network by ensuring that all nodes on the network agree on the ordering of the blocks in the chain. | checked(sven) | - |
+| 区块详情页 | Timestamp | In CKB, Timestamp refers to the timestamp of a block, which represents the time at which the block was created by a miner. The Timestamp is used to order the blocks in the blockchain. | checked(sven) | - |
```

Comment:
`The Timestamp is used to order the blocks in the blockchain`. I'm not sure `Timestamp` is used for that.

Reply:
> `The Timestamp is used to order the blocks in the blockchain`. I'm not sure `Timestamp` is used for that.

It's not correct, blocks are ordered by block number and connected by `parent hash` in block header.

The timestamp in the block header is an arbitrary value filled by the miner. The offset of timestamp will be verified by other nodes during broadcasting but should not be used as the order, it's just a representative attribute of a block.

Besides, `block number + parent hash` is good to order blocks because they can be used to make sure blocks are consecutive.